tests/vm_state_test.go

```diff
@@ -28,9 +20,14 @@ import (
 
 	"kubevirt.io/kubevirt/tests/console"
 	"kubevirt.io/kubevirt/tests/decorators"
+	"kubevirt.io/kubevirt/tests/framework/matcher"
```

Comment:
```suggestion
	. "kubevirt.io/kubevirt/tests/framework/matcher"
```

nit: Makes it easier to use the matchers.

Reply:
So normally dot imports are discouraged by linters
`should not use dot imports (ST1001)go-staticcheck`

Reply:
Does it make `make lint` fail? Dot-importing this package is done a lot in the test code base and at least in tests I think it is fine (like for ginkgo/gomega).

Reply:
It doesn't, since we probably opt out of this linter. You can find occurrences of both imports through the tests package